src/libs/OptionsListUtils.js

```diff
@@ -204,20 +205,76 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (!_.contains(participants, report.ownerEmail)) {
+            participants.push(report.ownerEmail);
+        }
+    }
+    return participants;
+}
+
+/**
+ *  Get the Avatar url or fallback to the default icon according to the chat type
+ *
+ * @param {String} source
+ * @param {Object} options
+ * @param {Boolean} [options.isChatRoom]
+ * @param {Boolean} [options.isArchivedRoom]
+ * @param {Boolean} [options.isPolicyExpenseChat]
+ * @returns {String | function():any}
```

Comment:
How about just `returns {*}`? I think most places we have standardized on using the capitalized docs. Not sure I have seen `function():any` used before or understand what it means 😬 

Caveat...

1. We have style guide recommendations here for JSDocs https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs
2. They could probably be better so please feel free to propose changes
3. Probably would be best to follow the style guide until we have done Step 2 to avoid blocking